client/homescreen/activity-panel/orders/index.js

```diff
@@ -361,10 +361,17 @@ export default withSelect( ( select, props ) => {
 	/* eslint-enable @wordpress/no-unused-vars-before-return */
 	let orders = [];
 
-	if ( reportOrders && reportOrders.length ) {
-		if ( actionableOrders && ! actionableOrders.length ) {
-			return { isRequesting: true };
-		}
+	if (
+		countUnreadOrders === null ||
+		typeof reportOrders === 'undefined' ||
+		typeof actionableOrders === 'undefined' ||
```

Comment:
Can `actionableOrders` be `undefined`?  Looks like an array is created above.

Reply:
Nice catch @joshuatf! We don't need that check. It was removed in the commit dcac5b1